app/controllers/catalog_controller.rb

```diff
@@ -181,6 +181,7 @@ def index
     config.add_index_field 'publication_display_ssm', label: 'Publication Statement', sr_only: true
     config.add_index_field 'edition_display_ssm', label: 'Edition', sr_only: true
     config.add_index_field 'full_links_struct', label: 'Access Online', helper_method: :generic_link
+    config.add_index_field 'hathitrust_struct', label: 'HathiTrust Access', helper_method: :hathi_link, hide_label: true
```

Comment:
Can you use the `sr_only` flag instead? I replaced the "hide label" functionality in this #580

Reply:
just tried with `sr_only` and it did not hide the label, is there sth else to do with that flag to hide the label?

Reply:
I see, ok. Well it should be working fine for the index view at least. The sr_only logic would have to be added to the show partials too.

Leaving the label out creates an accessibility problem and a semantic web problem. You'd have a `dl` child item with only a `dd` and no `dt` in cases of "hiding the label."

Reply:
ahh ok, I will add that logic to show as well then use it.